addons/website_slides/controllers/main.py

```diff
@@ -516,6 +520,14 @@ def slide_channel_tag_search_read(self, fields, domain):
             'can_create': can_create,
         }
 
+    @http.route(['/slides/channel/subscribe'], type='json', auth='public', website=True)
+    def slide_channel_subscribe(self, channel_id):
```

Comment:
I would return True in all those new routes ... just to return something that is not None.